src/meta/src/hummock/manager/mod.rs

```diff
@@ -1020,6 +1038,7 @@ where
         context_id: Option<HummockContextId>,
         compact_task: &mut CompactTask,
         compaction_guard: Option<RwLockWriteGuard<'_, Compaction>>,
+        table_stats_change: Option<TableStatsMap>,
```

Comment:
Is it possible that stat changes in `sync_group`?

Reply:
I think `sync_group` doesn't affect table stats.
When `sync_group` removes a compaction group, its tables are dropped without going through compaction. Those table's stat will be cleaned by `purge_table_stats` later asynchronously.
When `sync_group` adds a compaction group, either due to split or not, table stats don't change.

Reply:
> I think `sync_group` doesn't affect table stats.
> When `sync_group` removes a compaction group, its tables are dropped without going through compaction. Those table's stat will be cleaned by `purge_table_stats` later asynchronously.
> When `sync_group` adds a compaction group, either due to split or not, table stats don't change.

I see.